cmd/minikube/cmd/config/addons_list.go

```diff
@@ -89,62 +89,58 @@ var stringFromStatus = func(addonStatus bool) string {
 }
 
 var printAddonsList = func(cc *config.ClusterConfig, printDocs bool) {
+	// Get and sort addon names
 	addonNames := make([]string, 0, len(assets.Addons))
 	for addonName := range assets.Addons {
 		addonNames = append(addonNames, addonName)
 	}
 	sort.Strings(addonNames)
 
+	// Create table with default formatting (matches "Simple Tables" example in docs)
 	table := tablewriter.NewWriter(os.Stdout)
-	table.SetAutoFormatHeaders(true)
-	table.SetBorders(tablewriter.Border{Left: true, Top: true, Right: true, Bottom: true})
-	table.SetCenterSeparator("|")
 
-	// Create table header
-	var tHeader []string
+	// Set table header
+	var headers []string
 	if cc == nil {
-		tHeader = []string{"Addon Name", "Maintainer"}
+		headers = []string{"Addon Name", "Maintainer"}
 	} else {
-		tHeader = []string{"Addon Name", "Profile", "Status", "Maintainer"}
+		headers = []string{"Addon Name", "Profile", "Status", "Maintainer"}
 	}
 	if printDocs {
-		tHeader = append(tHeader, "Docs")
+		headers = append(headers, "Docs")
 	}
-	table.SetHeader(tHeader)
+	table.Header(headers)
 
-	// Create table data
-	var tData [][]string
-	var temp []string
+	// Prepare table data
+	var data [][]string
 	for _, addonName := range addonNames {
-		addonBundle := assets.Addons[addonName]
-		maintainer := addonBundle.Maintainer
+		addon := assets.Addons[addonName]
+		maintainer := addon.Maintainer
 		if maintainer == "" {
 			maintainer = "3rd party (unknown)"
 		}
-		docs := addonBundle.Docs
-		if docs == "" {
-			docs = "n/a"
-		}
+
+		var row []string
 		if cc == nil {
-			temp = []string{addonName, maintainer}
+			row = []string{addonName, maintainer}
 		} else {
-			enabled := addonBundle.IsEnabled(cc)
-			temp = []string{addonName, cc.Name, fmt.Sprintf("%s %s", stringFromStatus(enabled), iconFromStatus(enabled)), maintainer}
+			enabled := addon.IsEnabled(cc)
+			row = []string{addonName, cc.Name, fmt.Sprintf("%s %s", stringFromStatus(enabled), iconFromStatus(enabled)), maintainer}
 		}
+
 		if printDocs {
-			temp = append(temp, docs)
+			row = append(row, addon.Docs)
 		}
-		tData = append(tData, temp)
+
+		data = append(data, row)
 	}
-	table.AppendBulk(tData)
 
+	// Add all data at once (matches Bulk example in docs)
+	table.Bulk(data)
 	table.Render()
 
-	v, _, err := config.ListProfiles()
-	if err != nil {
-		klog.Errorf("list profiles returned error: %v", err)
-	}
-	if len(v) > 1 {
+	// Show profile tip if needed
+	if profiles, _, err := config.ListProfiles(); err == nil && len(profiles) > 1 {
```

Comment:
@Victorthedev why Error handling here was Removed? dont remove that code, we need to check for every eerror and handle it properly